searchInRotatedArr.js

```diff
@@ -0,0 +1,63 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/search-in-rotated-sorted-array/
+
+const search = function (nums, target) {
+  let pointer1 = 0;
+  let pointer2 = nums.length - 1;
+  if (nums[pointer1] === target) {
+    return pointer1;
+  }
+  if (nums[pointer2] === target) {
+    return pointer2;
+  }
+  let left = nums;
+  let right = [];
```

Comment:
We can define these 2 vars closer to where they are actually being used (move it couple of lines down).